addons/web/static/src/core/model_field_selector/unique_popover_hook.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { usePopover } from "@web/core/popover/popover_hook";
+
+export function useUniquePopover() {
```

Comment:
Why not add `export` in front of the original function ? Why move it in its own file ? Just to be clear, I'm not necessarily _against_ it, I'm just wondering.